README.md

```diff
@@ -6,6 +6,23 @@ Our electricity is not made equal and can be associated with higher or lower car
 
 The web app and the underlying data pipeline was built using open source tools. Web app's front-end was built using Shiny, electricity composition was extracted from National Grid's [Carbon Intensity API](https://carbonintensity.org.uk/) using [{intensegRid}](https://kkulma.github.io/intensegRid/articles/intro-to-carbon-intensity.html) package. Data and deployment pipelines were developed using GitHub Actions: `schedule.yml` pulls the carbon intensity data from the API, saves it in `data/daily_ci.rds` file daily and pushes the changes to the repo; `deploy.yml` deploys the changes to the [shinyapps.io server](https://kasiakulma.shinyapps.io/carbon-intensity-app/) on push. 
 
+## Getting start
```

Comment:
could you please correct the typo (to Getting start**ed**)?

Reply:
PR was updated accordingly.